model/src/main/resources/reference.conf

```diff
@@ -13,7 +22,7 @@ ort {
         }
 
         localFileStorage {
-          directory = ~/.ort/scanner
+          directory = ~/.ort/scanner/results
```

Comment:
Shouldn't this be part of https://github.com/heremaps/oss-review-toolkit/pull/1899?

Reply:
Added to #1899, I will rebase this PR later.